parm/config/gefs/config.ufs

```diff
@@ -72,14 +72,18 @@ fi
 case "${fv3_res}" in
     "C48")
         export DELTIM=1200
+        if [[ "${RUN}" = *gfs ]]; then
+          export layout_x=1
+          export layout_y=1
+          export nthreads_fv3=1
+          export nthreads_ufs=1
+          export WRITE_GROUP=1
+          export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
+        fi
         export layout_x=1
         export layout_y=1
-        export layout_x_gfs=1
-        export layout_y_gfs=1
         export nthreads_fv3=1
-        export nthreads_fv3_gfs=1
         export nthreads_ufs=1
-        export nthreads_ufs_gfs=1
```

Comment:
I think you don't need a `if` block here since the resources are identical for `gdas` and `gfs` RUNs.

```suggestion
        export layout_x=1
        export layout_y=1
        export nthreads_fv3=1
        export nthreads_ufs=1
        export WRITE_GROUP=1
        export WRTTASK_PER_GROUP_PER_THREAD_PER_TILE=1
```